### ISSUE TITLE
Please update the cache the docker image of Windows Server 2016 LTSC

### ISSUE THREAD

**Issue by eriawan** [Microsoft/azure-pipelines-image-generation#536]
# Update request description

The current image of Docker images cache in the hosted image  of latest Windows Server 2016 LTSC images are not synchronized with latest Docker hub image of Windows Server 2016 LTSC on VM pool VS2017-Win2016.

As far as I know, the `microsoft/windowsservercore:latest` is always having the same SHA1 as the `microsoft/windowsservercore:ltsc2016` by looking at both the Microsoft's docker hub repo and checking the docker image repo on the cached VM image itself.

The image ID and the digests are not up to date. For example, this is the screenshot today before update:

![image](https://user-images.githubusercontent.com/8773147/49865612-0593dc00-fe38-11e8-9aff-0190f0f09377.png)

and after update:
![image](https://user-images.githubusercontent.com/8773147/49867338-c1efa100-fe3c-11e8-8484-98443ffeee77.png)

On my Azure DevOps build log:

![image](https://user-images.githubusercontent.com/8773147/49867228-68877200-fe3c-11e8-9881-0897624f8935.png)

It shows that the image is not in synched, and the last sync is 3 months ago. 

## Tool info
- Tool name: 
- Add or update? Update the image to contain cached Windows Server 2016 LTSC
- Desired version: 
- If this is an add request:
  - Brief description of tool: 
  - URL for tool's homepage: 
  - Open source license type (Apache, MIT, GPL, etc.): 

## Areas and ecosystems

- [ ] Android
- [ ] C / C++
- [x] Container tools
- [ ] Database servers or clients
- [ ] General purpose tools
- [ ] Go
- [ ] Java
- [ ] Javascript, Node.js, and TypeScript
- [ ] iOS, macOS, and Apple-specific tools
- [ ] .NET Core or .NET Desktop
- [ ] PHP
- [ ] Python
- [ ] Ruby
- [ ] Rust
- [ ] Scala
- [ ] Testing and code coverage
- [ ] Xamarin
- [ ] Other

## Hosted pools

- [ ] macOS
- [ ] Ubuntu
- [x] Windows

**Comment by eriawan** [comment]
Additional information: the SHA1 of both `microsoft/windowsservercore:latest` and `microsoft/windowsservercore:ltsc2016` should be:
`efd8555802bd74561e0ec7652ca9bfbe8d4846e2ed4bf5b1fcf5d3048767a82a` and it's updated yesterday (as shown as "15 hours ago").

Therefore the cached image on the current VM of VS2017-Win2016 is not the latest.

**Comment by DavidStaheli** [comment]
@vtbassmatt is this still something your team would look at?

**Comment by vtbassmatt** [comment]
I think so. @lokesh755

**Comment by eriawan** [comment]
@DavidStaheli @lokesh755 

Is there any updats on this issue?

**Comment by lokesh755** [comment]
There will be an image update in first week of Jan

**Comment by eriawan** [comment]
@lokesh755 
Is the image update available now?

Please confirm.

**Comment by vtbassmatt** [comment]
We ran into an unexpected delay deploying the new image. I'm sorry for the frustration this is causing you (and other customers). @joebourneMS is working to get this unblocked as quickly as possible. We _hope_ to get it out next week (Jan 14-18) but that isn't a firm commitment until we finish our investigation.

**Comment by eriawan** [comment]
@vtbassmatt 
see also my issue on #499 
I think it needs update ASAP, otherwise the image of VS2017 won't be usable for my dev team because it's too outdated and can't keep up with VS 2017 15.9.x releases at all.

Any progress on this? it's nearing a week...

**Comment by vtbassmatt** [comment]
I understand the pain this is causing you, and I'm really sorry. As Lokesh mentioned, we've been hit by a string of issues getting this update out. We hope to have it out this week.

**Comment by eriawan** [comment]
@DavidStaheli @vtbassmatt @lokesh755 

I have tested today, and now the image has been updated to match the latest Microsoft's `microsoft/dotnet-framework:4.7.2-runtime-windowsservercore-ltsc2016` image.

Current SHA256 as of today (11th February, 2019) on my local machine:

`sha256:69871ca8f38dfc1f9a52f2b2b35112a71031d6f8ce325cc31e47af9eb714d8fa`

![image](https://user-images.githubusercontent.com/8773147/52551206-02a07200-2e0e-11e9-99af-7372298674bf.png)

and on Azure DevOps:

![image](https://user-images.githubusercontent.com/8773147/52551238-2e235c80-2e0e-11e9-94a9-735e1b67d842.png)

So now my build time is quicker! Thanks guys!
I'm closing this issue for now.
